installer/README.md

```diff
@@ -28,6 +28,31 @@ go get github.com/GoogleCloudPlatform/k8s-service-catalog/installer/cmd/sc
 
 After running the above command, `sc` should get installed in your GOPATH/bin dir.
 
+## Requirements
```

Comment:
we should move it before installation I guess.

Reply:
Pl. change the order of the links at the top as well.

Reply:
Done. Thanks!